default-recommendations/conditional-suggestions-test.rkt

```diff
@@ -0,0 +1,33 @@
+#lang resyntax/testing/refactoring-test
+
+
+require: resyntax/default-recommendations conditional-suggestions
+
+
+test: "if-else *doesn't* refactor to cond"
+------------------------------
+#lang racket/base
+(if 'cond 'then 'else)
+------------------------------
+
+
+test: "singly-nested if-else *doesn't* refactor to cond"
+------------------------------
+#lang racket/base
+(if 'cond 'then (if 'cond2 'then2 'else))
+------------------------------
+
+
+test: "if-else chain refactors to cond"
```

Comment:
I think it would be good to have a second test that's just like this one but where the input ifs are spread across multiple lines. Suggested naming: "multi-line if-else chain refactors to multi-line cond" for the new test, and change this existing test's name to "one-line if-else chain refactors to multi-line cond".